boards/arm/mimxrt1010_evk/mimxrt1010_evk.dts

```diff
@@ -52,11 +52,12 @@ arduino_serial: &lpuart1 {};
 
 &flexspi {
 	reg = <0x402a8000 0x4000>, <0x60000000 DT_SIZE_M(16)>;
-	at25sf128a: at25sf128a@0 {
+	at25sf128a: at25sf128a@60000000 {
```

Comment:
I tried this before and got a NAK from @galak; he felt strongly that this address should be the SPI chip select. See #30065. I really would like to get rid of the CODE_LOCATION stuff though.

Reply:
Yeah, I probably have the same NAK as before.

Reply:
See my comment below- I agree with @MaureenHelm that there would be real value to removing the `CODE_LOCATION` Kconfig values, and I'm happy to change the approach we are taking here to preserve the SPI chip select values. The main priority is enabling the use of `zephyr,flash` to select the code location, to simplify changing the code location on our platforms for customers.